sympy/core/__init__.py

```diff
@@ -48,6 +49,8 @@
 
     'S',
 
+    'Equation', 'Eqn',
```

Comment:
I suggest introducing `sympy/equation` module rather than defining this in `sympy/core`. `core` should contain "core" features which are imported by other modules. `Eqn` does not really fit into here because it's user-oriented.

Reply:
The reason it was put into core is that many of us want to use and return equations as part of Solve.

Reply:
That does not require `Eqn` to be in core since `sympy.solvers` can import `sympy.equation`.

Reply:
I am definitely not wedded to it being in core. I put it there at other peoples' suggestion. From the comments, I believe core was also suggested because people think of the concept of an equation as a core concept in mathematics. Pulling it out into its own module allows some more flexibility for organization of the code into separate files as well.